package.json

```diff
@@ -5,22 +5,31 @@
     "typings": "./dist/typings/index.d.ts",
     "main": "dist/commonjs/index.js",
     "types": "dist/typings/index.ts",
+    "bin": {
+        "fusebox": "bin/fsbx"
+    },
     "directories": {
+        "bin": "bin",
         "test": "test",
         "modules": "modules",
         "lib": "dist/commonjs",
-        "typings": "dist/typings"
+        "typings": "dist/typings",
+        "docs": "docs"
     },
     "files": [
+        "bin",
         "src",
         "dist",
         "modules",
         "test",
         "docs"
     ],
     "scripts": {
+        "test-cli": "fusebox",
         "help": "node bin/fsbx --help",
-        "test": "TRAVIS=true ./node_modules/mocha/bin/mocha && node ./test.js",
+        "lint-fix": "eslint src/*.ts src/**/*.ts src/**/**/*.ts test/*.js test/**/*.js --fix",
```

Comment:
got it. can we just config the parser to be [typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser), as it is ESTree-compatible?